src/cascadia/TerminalApp/AppKeyBindings.cpp

```diff
@@ -47,327 +47,327 @@ namespace winrt::TerminalApp::implementation
         {
         case ShortcutAction::CopyText:
         {
-            auto args = CopyTextArgs();
+            CopyTextArgs args{};
             args.TrimWhitespace(true);

```

Comment:
I'm surprised this setter actually works/exists. How?? It's not in the idl, and you're using the projected type (instead of the internal one)

Reply:
Also, you _should_ consider using the impl type using `winrt::make_self<xxxx::CopyTextArgs::implementation>()`

Reply:
It's because we're in the `TerminalApp::implementation` namespace, not the `TerminalApp` namespace. So we're making a `TerminalApp::implementation::CopyTextArgs`, not a projected type

Reply:
In that case this is technically a very bad thing! We should never stack-allocate implementation types. They _must_ be heap allocated

Reply:
wait _really_? Why does this seem to work then?